app/presenters/candidate_interface/application_form_presenter.rb

```diff
@@ -88,6 +88,14 @@ def other_qualification_path
       end
     end
 
+    def volunteering_path
+      if volunteering_completed? || volunteering_added? || [true, false].include?(@application_form.volunteering_experience)
```

Comment:
What can `@application_form.volunteering_experience` be if not true or false, `nil?`

Reply:
Yeah, so when it's `nil` then we know the candidate has arrived at the volunteering section for the first time.

Reply:
Ended up doing this, means we don't have to update the attribute in `VolunteeringRoleForm` and removes the need for this monstrosity `[true, false].include?(@application_form.volunteering_experience)`.

Reply:
Nvm, spoke to @tvararu. We're reverting back to original.